egs/mgb5/s5/local/score.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
```

Comment:
unless you modified it, can you make it a symlink?

Reply:
I will modify it to use different transcription and multi-reference WER. Still waiting for ELRA to fix alignment. So, I prefer to have a local copy.

Reply:
do not forget about this